server/database/config/db_connection.js

```diff
@@ -0,0 +1,27 @@
+const { Pool } = require('pg');
+require('dotenv').config();
+const url = require('url');
+
+let DB_URL = '';
+if (process.env.NODE_ENV === 'test') {
+  DB_URL = process.env.DATABASE_TESTING;
+} else if (process.env.NODE_ENV === 'dev') {
+  DB_URL = process.env.DATABASE_LOCAL;
+} else if (process.env.NODE_ENV === 'production') {
+  DB_URL = process.env.DATABASE_HEROCO;
+} else {
+  throw new Error('not found DB_URL');
+}
+const params = url.parse(DB_URL);
+const [user, password] = params.auth.split(':');
+const options = {
```

Comment:
when you have variable have the same of the variable you should put it at the top of the object
```
const options = {
  user,
  password,
  host: params.hostname,
  port: params.port,
  database: params.path.split('/')[1],
  ssl: process.env.host !== 'localhost',
  max: process.env.DB_MAX_CONNECTIONS || 2
};
```